scale/scale/settings.py

```diff
@@ -29,6 +29,9 @@
 # The full name for the Scale Docker image (without version tag)
 SCALE_DOCKER_IMAGE = 'geoint/scale'
 
+# The location of the config file containing Docker credentials
+CONFIG_URI = os.environ.get('CONFIG_URI', None)
```

Comment:
Don't use env vars in the settings.py. This should be a default value only so change this to `CONFIG_URL = None`